src/components/modules/shared/ArticleCountOptOut2.tsx

```diff
@@ -0,0 +1,153 @@
+// --- NB --- //
+// This is a temporary component whilst we're running an article count
+// A/B test. The aim is to deprecate this type of opt out in favour of
+// one with better UX.
```

Comment:
We will delete this! This is a quick hack becuase one of the new designs isn't quite compatible. We can remove this as soon as we've finished the design test and implement the new UX.